Formula/browserpass.rb

```diff
@@ -0,0 +1,54 @@
+class Browserpass < Formula
+  desc "Native component for Chrome & Firefox password management add-on"
+  homepage "https://github.com/dannyvankooten/browserpass"
+  url "https://github.com/dannyvankooten/browserpass/archive/2.0.8.tar.gz"
+  sha256 "1747fc24e51128811eb679b43ab23a8cce6ac566620112f0ca9fa129ace4a928"
+
+  depends_on "dep" => :build
+  depends_on "go" => :build
+  depends_on "node" => :build
+  depends_on "yarn" => :build
+
+  def install
+    ENV["GOPATH"] = buildpath
+    (buildpath/"src/github.com/dannyvankooten/browserpass").install buildpath.children
+    cd "src/github.com/dannyvankooten/browserpass" do
+      system "make", "deps"
+      system "make", "js"
+      system "make", "browserpass-darwinx64"
```

Comment:
you can combine all 3 goals in one system call, "make deps js browserpass-darwinx64"

Reply:
Doesn't compile when I combine them. Go complains about not being able to find some packages (`twofactor` and `go-zglob`).

Reply:
Good point, actually I forgot to tell you, you should download archive from a different link [1], that one already contains all the dependencies so that you don't need to do `make deps` and `make js` at all. Then you only need to do `make browserpass-darwinx64`. That will also allow you to remove dependencies on "dep", "yarn" and "node".

[1]: https://github.com/dannyvankooten/browserpass/releases/download/2.0.8/browserpass-src.tar.gz